src/elements/Tree.tsx

```diff
@@ -0,0 +1,321 @@
+import * as React from "react";
+import clsx from "clsx";
+import Box, { BoxProps } from "@vertigis/web/ui/Box";
+import Checkbox from "@vertigis/web/ui/Checkbox";
+import ChevronDownIcon from "@vertigis/web/ui/icons/ChevronDown";
+import Collapse from "@vertigis/web/ui/Collapse";
+import DynamicIcon, { DynamicIconProps } from "@vertigis/web/ui/DynamicIcon";
+import { FormElementProps, FormElementRegistration } from "@geocortex/workflow/runtime";
+import IconButton, { IconButtonProps } from "@vertigis/web/ui/IconButton";
+import Link from "@vertigis/web/ui/Link";
+import List, { ListProps } from "@vertigis/web/ui/List";
+import ListItemIcon from "@vertigis/web/ui/ListItemIcon";
+import ListItemText from "@vertigis/web/ui/ListItemText";
+import ListItemButton from "@vertigis/web/ui/ListItemButton";
+import { UIContext } from "@vertigis/web/ui/UIContext";
+import { BrandingService } from "@vertigis/web/branding/BrandingService";
+/**
+ * Properties for the `Tree` component.
+ */
+export declare type TreeViewProps = {
```

Comment:
This and `SettableCheckBoxProps` on line 37 seems unnecessary. Why not just put `showCheckBoxes?: boolean;` on line 58?

Lines 33 and 35 are grapping props from other components. We don't need to match that pattern for the component we are creating in this file.

Reply:
Removed